test/components/helpers/component-testing.coffee

```diff
@@ -38,11 +43,12 @@ Testing = {
       props = _.clone(options.props)
       props._wrapped_component = component
       wrapper = ReactTestUtils.renderIntoDocument React.createElement(Wrapper, props)
-      resolve({
+      RENDERED.push(wrapper)
+      resolve(
         wrapper,
         element: wrapper.refs.element,
         dom: React.findDOMNode(wrapper.refs.element)
-      })
+      )
 
```

Comment:
A puzzler.  Here i'm resolving a promise with the rendered element which works fine.  However I also need to unmount the component after all of the promise's `then` callbacks are complete.  I can't seem to figure out how to do so or even if it's possible.  Taking a `.then()` onto the `resolve({...})` here doesn't cause any errors but it's seemingly never called.